cime_config/config_compsets.xml

```diff
@@ -202,7 +227,7 @@
   </compset>
   <compset>
     <alias>I1850Clm60BgcCropCmip6</alias>
-    <lname>1850_DATM%GSWP3v1_CLM60%BGC-CROP-CMIP6DECK_SICE_SOCN_MOSART_SGLC_SWAV</lname>
+    <lname>1850_DATM%CRUJRA2024_CLM60%BGC-CROP-CMIP6DECK_SICE_SOCN_MOSART_SGLC_SWAV</lname>
```

Comment:
@ekluzek did I jump the gun here and in other cmip compsets? Should I have waited to do this in the to-be-done-later answer-changing PR?

Reply:
I reverted the cmip compsets to gswp3 for b4b. Instead of putting a reminder in #2961, I added corresponding explicit crujra compsets. These will later replace the current default cmip compsets.